libs/@guardian/ab-core/eslint.config.js

```diff
@@ -0,0 +1,14 @@
+import guardian from '@guardian/eslint-config';
+
+export default [
+	...guardian.configs.recommended,
+	...guardian.configs.jest,
+	{
+		ignores: [
+			'node_modules',
```

Comment:
This is ignored automatically by eslint, can see the argument for writing it anyway